docs/kubernetes-installers.md

```diff
@@ -0,0 +1,63 @@
+# K8s Installers and Distributions
+
+## Tested installers and distributions
+
+The table below is not comprehensive. Antrea should work with most K8s
+installers and distributions. The table refers to specific version combinations
+which are known to work and have been tested, but support is not limited to that
+list. Each Antrea version supports [multiple K8s minor
+versions](versioning.md#supported-k8s-versions), and installers / distributions
+based on any one of these K8s versions should work with that Antrea version.
+
+| Antrea Version | Installer / Distribution | Cloud Infra | Node Info | Node Size | Conformance Results | Comments |
+|-|-|-|-|-|-|-|
+| v1.0.0 | Kubeadm v1.21.0 | AWS EC2 | Ubuntu 20.04.2 LTS (5.4.0-1045-aws) amd64, docker://20.10.6 | t3.medium |  |  |
+|  |  |  | Windows Server 2019 Datacenter (10.0.17763.1817), docker://19.3.14 | t3.medium |  |  |
+|  |  |  | Ubuntu 20.04.2 LTS (5.4.0-1045-aws) arm64, docker://20.10.6 | t3.medium |  |  |
+|  | Rancher v2.5.7, K8s v1.20.5-rancher1-1 | AWS EC2 | Ubuntu 18.04.5 LTS (5.4.0-1045-aws) amd64, docker://19.3.15 | t3a.medium | Some tests failing because of [this issue](https://github.com/kubernetes/kubernetes/pull/100199) |  |
+|  | K3s v1.19.8+k3s1 | [OSUOSL] | Ubuntu 20.04.1 LTS (5.4.0-66-generic) arm64, containerd://1.4.3-k3s3 | 2 vCPUs, 4GB RAM | Installed with [k3sup] 0.9.13 | Antrea CI |
```

Comment:
The "Conformance Results" of K3s is "Installed with [k3sup] 0.9.13"?

Reply:
Thanks, typo